Docker/Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
 	apt-get install --no-install-recommends -y \
 	ca-certificates cron \
-	apache2 libapache2-mod-php \
+	apache2 libapache2-mod-fcgid php-fpm \
```

Comment:
I am not sure of the exact pros and cons for the FreshRSS main use-cases, but https://cwiki.apache.org/confluence/display/HTTPD/Php-fcgid tells:

> On what conditions not to run php on fcgid
> If you run httpd 2.4, you should consider [PHP-FPM](http://wiki.apache.org/httpd/PHP-FPM)

And https://cwiki.apache.org/confluence/display/httpd/PHP-FPM tells:

> using mod_proxy_fcgi and php-fpm